README.md

```diff
@@ -54,29 +54,6 @@ By specifying `server_uri`, the fleetadapter will update `rmf-web` `api-server`
 * [Campus World](#Campus-World)
 * [Manufacturing & Logistics World](#Manufacturing-&-Logistics-World)
 
-**RMF Panel**
```

Comment:
Can instead direct users to `rmf-web` here?

Reply:
`rmf-web` is already introduced just before this paragraph, so that would be a bit redundant